velero-plugin-for-aws/object_store.go

```diff
@@ -234,6 +239,12 @@ func (o *ObjectStore) Init(config map[string]string) error {
 		o.preSignS3 = o.s3
 	}
 
+	if tagging != "" {
+		if CheckTags(tagging) != nil {
```

Comment:
Do we want to return the error from `CheckTags`? If so, shouldn't this line be something like:
```
err = CheckTags(tagging)
if err != nil {
   return err
}
```

Reply:
@mdnahian I have addressed it.